src/main/scala-2.12/org/apache/james/gatling/imap/scenari/ImapFullScenario.scala

```diff
@@ -25,6 +25,9 @@ class ImapFullScenario {
 
   val readLastEmail = exec(imap("list").list("", "*").check(ok, hasFolder("INBOX")))
     .exec(imap("select").select("INBOX").check(ok))
+    // The imapnio library fail to parse the attributes BODY[TEXT] and BODY[HEADER]
+    // so for the time being we will only fetch the BODYSTRUCTURE
+    // the corresponding issue can be found at :  https://ci.linagora.com/linagora/lgs/openpaas/james/issues/1243
```

Comment:
agree, but ci.linagora.com is not available for external users, so please create a github issue and reference it inside